gui/src/components/Sidebar.svelte

```diff
@@ -109,16 +109,39 @@
       disabled={btnDisabled}
       class:is-disabled={btnDisabled}>{buttonText}</button
     >
+    {#if selectedVersion && selectedVersion.installed}
+      <button
+        type="button"
+        class="nes-btn is-warning play-btn"
+        on:click={() => {
+          invoke("uninstall", {
+            version: selectedVersion.version,
+          }).then(() => {
+            selectedVersion.installed = false;
+            $downloadProgress.event = "Finished";
+          });
+        }}
+        disabled={btnDisabled}
+        class:is-disabled={btnDisabled}>Uninstall</button
+      >
+    {/if}
```

Comment:
Haven't tried running this but having a review from @bufgix would be nice.

Reply:
Maybe we can move the uninstall function to the `<script>` tag then call in the html. Also, if we use async function with `async/await` keyword instead of `.then()/.cath()` would be great

Reply:
There you go. 👍